content/en/docs/reference/glossary/Preemption.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Preemption
+id: Preemption
+date: 2019-01-31
+full_link: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#preemption
+short_description: >
+  Preemption logic in Kubernetes helps a pending Pod find a suitable node by deleting low priority Pods existing on that node.
```

Comment:
we can slight change this sentence.

>  Preemption logic in Kubernetes helps a pending Pod to find a suitable Node by evicting low priority Pods existing on that node.

sorry i didn't catch this earlier.
[1]